### PR TITLE
feat: DAH-2609 change text and link destination for need help block

### PR DESCRIPTION
**Pull request by tallulahkay**
## Description

Updates text and link destination for Need Help component.

## Jira ticket

https://sfgovdt.jira.com/browse/DAH-2609

### Version Control

- [ ] branch name begins with `angular` if it contains updates to Angular code
- [x] branch name contains the Jira ticket number
- [x] PR name follows `type: TICKET-NUMBER Description` format, e.g. `feat: DAH-123 New Feature`

### Code quality

- [x] [the set of changes is small](https://google.github.io/eng-practices/review/developer/small-cls.html#what-is-small)
- [ ] all automated code checks pass (linting, tests, coverage, etc.)
- [x] code irrelevant to the ticket is not modified e.g. changing indentation due to automated formatting
- [x] if the code changes the UI, it matches the UI design exactly

### Review instructions

- [x] instructions specify which environment(s) it applies to
- [x] instructions work for PA testers
- [x] instructions have already been performed at least once

### Request review

- [x] PR has `needs review` label
- [x] Use `Housing Eng` group to automatically assign reviewers, and/or assign specific engineers
- [ ] If time sensitive, notify engineers in Slack

### CODE REVIEW

File: app/javascript/modules/listingDetailsAside/ListingDetailsAside.tsx [hunk 1]
```diff
@@ -34,28 +34,6 @@ export const ListingDetailsAside = ({ listing, imageSrc }: ListingDetailsSidebar
     </SidebarBlock>
   )
```
Comment (cade-exygy) [inline app/javascript/modules/listingDetailsAside/ListingDetailsAside.tsx:36]: **question (non-blocking):**  This is just a refactor right? Not related to How to Apply AC. Not a concern, just confirming!

Comment (cade-exygy) [reply app/javascript/modules/listingDetailsAside/ListingDetailsAside.tsx:36]: Screenshot 2024-11-13 at 10 38 23 AM
Style issues on rental and sales listing details. 

Happy to pair on this, the styles for the new GetHelpSidebar block are probably conflicting with something on the ListingDetailsAside

**Review by cade-exygy — CHANGES_REQUESTED** [review]

File: app/javascript/modules/listingDetailsAside/ListingDetailsAside.tsx (hunk 1 above)
Comment (tallulahkay) [reply app/javascript/modules/listingDetailsAside/ListingDetailsAside.tsx:36]: Hi yes sorry I'm still working on this! Yeah I just want to be able to use the same component in both places so I don't have to recreate the wheel

**Comment by tallulahkay** [comment]
@cade-exygy I updated to a simpler implementation because the styling of the two pages are too different

**Review by cade-exygy — APPROVED** [review]
Looks good! Only question: On Mobile view, I am not seeing the Get Help block. Is it supposed to be visible?

**Comment by tallulahkay** [comment]
> Looks good! Only question: On Mobile view, I am not seeing the Get Help block. Is it supposed to be visible?

Mobile view of which type of listing?

**Comment by cade-exygy** [comment]
> > Looks good! Only question: On Mobile view, I am not seeing the Get Help block. Is it supposed to be visible?
> 
> Mobile view of which type of listing?

Non FCFS Sales was the one I was looking at. May apply to rental too

**Comment by tallulahkay** [comment]
> > > Looks good! Only question: On Mobile view, I am not seeing the Get Help block. Is it supposed to be visible?
> > 
> > 
> > Mobile view of which type of listing?
> 
> Non FCFS Sales was the one I was looking at. May apply to rental too

oops, thanks! That's AC I missed!

**Review by cade-exygy — APPROVED** [review]
Looks good!

**Pull request merged**